src/Symfony/Component/Serializer/Encoder/DecoderInterface.php

```diff
@@ -36,7 +36,7 @@ interface DecoderInterface
      *
      * @throws UnexpectedValueException
      */
-    public function decode($data, $format, array $context = []);
+    public function decode($data, string $format, array $context = []);
```

Comment:
Either we can type-hint `string $data` or the docblock is a lie. 🤔

Reply:
done :)